src/test/java/de/tracetronic/jenkins/plugins/ecutest/extension/warnings/WarningsRecorderIT.java

```diff
@@ -41,7 +41,7 @@ public boolean perform(final AbstractBuild<?, ?> build, final Launcher launcher,
             }
         });
 
-        final FreeStyleBuild build = jenkins.buildAndAssertStatus(Result.FAILURE, project);
+        final FreeStyleBuild build = jenkins.buildAndAssertStatus(Result.SUCCESS, project);
```

Comment:
How can it be that such changes do not influence the test result?

Reply:
Because WarningsNG plugin is now compatible with updated Jenkins baseline and does not fail anymore.